### PR TITLE
[RFR] Add AggregateStatusCard widget and tests

### PR DESCRIPTION
**Pull request by mshriver**
Adding widgets for Aggregate Status cards, per patternfly reference markup.

Making the locators modular, so that modifying them in application-specific widgets is easier.

### CODE REVIEW

File: testing/testing_page.html [hunk 1]
```diff
@@ -70,55 +70,175 @@ <h1>Widgetastic About Modal</h1>
...
-    <input class="form-control" id="date_readonly" name="date_readonly" readonly="readonly" autoclose="true" placeholder="MM/DD/YYYY" data-date-format="mm/dd/yyyy" type="text"/>
-  </div>
-     
-  <div class="datepicker_readwrite">
-    <label class="control-label " for="date_readwrite">
-      Bootstrap datepicker readwrite
-    </label>
-    <input class="form-control" id="date_readwrite" name="date_readwrite" autoclose="true" placeholder="MM/DD/YYYY" data-date-format="mm/dd/yyyy" type="text"/>
-  </div>
+  <div class="datepicker_container" align="center" style="width: 300px; text-align:center">
```
Comment (mshriver) [inline testing/testing_page.html]: This should all just be indentation changes, to get this in-line with the body element. I am not intending to change the datepicker widget on the test page.

File: src/widgetastic_patternfly/__init__.py [hunk 2]
```diff
@@ -2027,3 +2027,102 @@ def is_displayed(self):
             :py:class:`bool`
         """
         return self.browser.is_displayed(self.textbox)
+
+
+class AggregateStatusCard(Widget):
+    """Widget for patternfly aggrgate status card, used in dashboard views
```
Comment (jawatts) [inline src/widgetastic_patternfly/__init__.py]: Typo in aggregate here

File: src/widgetastic_patternfly/__init__.py [hunk 3]
```diff
@@ -2027,3 +2027,102 @@ def is_displayed(self):
...
+    This covers the standard type card
+    https://www.patternfly.org/pattern-library/cards/aggregate-status-card/#example-code-1
+    """
+    # Get the aggregate-status card div, per patternfly reference markup
+    ROOT = ParametrizedLocator(
+        './/div[contains(@class, "card-pf-aggregate-status") '
+        'and not(contains(@class, "card-pf-aggregate-status-mini")) '
+        'and h2[contains(@class, "card-pf-title")]'
+        '//span[normalize-space(following::text())={@name|quote}]]'
+    )
```
Comment (jawatts) [inline src/widgetastic_patternfly/__init__.py]: :+1:

File: src/widgetastic_patternfly/__init__.py [hunk 4]
```diff
@@ -2027,3 +2034,204 @@ def is_displayed(self):
...
+            return []
+        return [
+            StatusNotification(parent=self, note_element=note, logger=self.logger)
+            for note in notes
+        ]
+
+    def read(self):
+        items = {
+            k: getattr(self, k)
+            for k in ['icon', 'count', 'name']
```
Comment (RonnyPfannschmidt) [inline src/widgetastic_patternfly/__init__.py]: just writing the full dict out here might be a little more readable and comprehensible

**Pull request merged**